src/main/java/io/vlingo/wire/fdx/outbound/rsocket/RSocketOutboundChannel.java

```diff
@@ -52,36 +54,43 @@ public void close() {
     this.clientSocket = null;
   }
 
+  private final Scheduler scheduler = new Scheduler();
+
   @Override
-  public void write(final ByteBuffer buffer) {
```

Comment:
@alexguzun I resolved a merge conflict here. Please have a look to make sure that I didn't miss anything. I believe it was only style changes introduced here in your commit but I could be wrong.